fearless_simd/src/core_arch/x86/sse2.rs

```diff
@@ -34,8 +34,7 @@ impl Sse2 {
 
     delegate! { arch:
         fn _mm_pause();
-        #[allow(clippy::not_unsafe_ptr_arg_deref)]
-        fn _mm_clflush(p: *const u8);
+        unsafe fn _mm_clflush(p: *const u8);
```

Comment:
It's not entirely clear if this should be unsafe, but [there is evidence](https://www.felixcloutier.com/x86/clflush) that this can page fault.